examples/ppo/ppo.py

```diff
@@ -131,15 +128,15 @@ def main(args):
     collector = make_collector_onpolicy(
         make_env=create_env_fn,
         actor_model_explore=actor_model,
-        args=args,
+        cfg=cfg,
         # make_env_kwargs=[
         #     {"device": device} if device >= 0 else {}
-        #     for device in args.env_rendering_devices
+        #     for device in cfg.env_rendering_devices
         # ],
     )
 
     recorder = transformed_env_constructor(
-        args,
+        cfg,
```

Comment:
since you are passing kwargs as `key=val` format at most places, maybe we should stick to that style here as well i.e. `cfg=cfg`